blog/models.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.urls import reverse
+
+
+class User(AbstractUser):
+    class Meta:
+        ordering = ["-username"]
+
+    def __str__(self):
+        return self.username
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="posts"
+    )
+    title = models.CharField(max_length=255)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
```

Comment:
Just for the future: it's better to rename it to created_datetime because when frontender expects just date, it'll display as it is, but you may return this field and date will be returned as well